src/views/v2/day/top-bar.php

```diff
@@ -21,6 +21,8 @@
 
 	<?php $this->template( 'day/top-bar/datepicker' ); ?>
 
+	<?php $this->template( 'components/top-bar/category-color-picker' ); ?>
```

Comment:
Doesn't need t change, but I'd have used do_entry_point here so the template doesn't get injected at all if CC isn't active. (rather than bail in the loaded template)

Reply:
I'll have to remember that, that would of simplified the code a lot.